pennylane/qnode.py

```diff
@@ -1132,29 +1117,25 @@ def _pd_analytic_var(self, param_values, param_idx, **kwargs):
 
                     if not np.allclose(A @ A, np.identity(A.shape[0])):
                         # make a copy of the original variance
-                        old = copy.deepcopy(e)
+                        new = qml.expval(qml.ops.Hermitian(A @ A, w, do_queue=False))
```

Comment:
Are the deep copies now redundant? Trying to work out why they are no longer needed.

Reply:
I think so. The only change made to `applicable_nodes` is changing the `return_type` which we undo at the end, and `new` simply replaces `e` in the graph for a while until we put `e` back.